### PR TITLE
Fix issue with compiling sources with high LOC

### PR DESCRIPTION
**Pull request by anuruddhal**
## Purpose
- Fix the issue with Compiling/Running a package with more than 32,765 lines
- Resolves https://github.com/ballerina-platform/ballerina-lang/issues/9984

### CODE REVIEW

File: tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/loc/LineOfCodeTest.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+import org.ballerinalang.launcher.util.BCompileUtil;
+import org.ballerinalang.launcher.util.BRunUtil;
+import org.ballerinalang.launcher.util.CompileResult;
+import org.ballerinalang.model.values.BInteger;
+import org.ballerinalang.model.values.BValue;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
```
Comment (Kishanthan) [inline tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/loc/LineOfCodeTest.java]: Test class needs test description and class comments.

File: tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/loc/LineOfCodeTest.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package org.ballerinalang.test.loc;
```
Comment (Kishanthan) [inline tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/loc/LineOfCodeTest.java]: License header is missing

Comment (anuruddhal) [reply tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/loc/LineOfCodeTest.java]: Added with https://github.com/ballerina-platform/ballerina-lang/pull/9987/commits/e83a124f8e8b1974e2b3e9c10e798c4bc4393f2b

File: tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/loc/LineOfCodeTest.java (hunk 1 above)
Comment (anuruddhal) [reply tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/loc/LineOfCodeTest.java]: Added with https://github.com/ballerina-platform/ballerina-lang/pull/9987/commits/e83a124f8e8b1974e2b3e9c10e798c4bc4393f2b

**Comment by lankavitharana** [comment]
version increment seems already done, hence no need to increment again, sorry for the confusion

**Pull request merged**